auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthzJwtService.java

```diff
@@ -152,6 +152,64 @@ public void authorizeSchemaWrite(String token, String keyspace, String table, Sc
     // Cannot perform authorization with a JWT token so just return
   }
 
+  /**
+   * Authorization for role management is not provided by JWTs so all authorization will be deferred
+   * to the underlying permissions assigned to the role the JWT maps to.
```

Comment:
Does this hold for case when the `AuthenticatedUser` has `usesTransitionalAuth() == false`?

Reply:
Yes, it's when they're true that a different authz mechanism needs to be available.